modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectManagementToolbar.tsx

```diff
@@ -60,6 +60,9 @@ export default function ObjectManagementToolbar({
 			isRootDescendantNode={isRootDescendantNode}
 			label={label}
 			onExternalReferenceCodeChange={(externalReferenceCode: string) => {
+				if (!setValues) {
+					return;
+				}
```

Comment:
Hey @diogomelo09, thanks for working on this! I see that, with this PR, we are preventing `setValues` from being called and the save still works! How can that be? Either that `setValues` is useless or something that should be set is not being set. The real investigation here is to find out why `setValues` is undefined when the user navigates to tab other then "Details". And also evaluate if the implications of not executing this `setValues`.